packages/mdc-drawer/README.md

```diff
@@ -331,7 +331,8 @@ Mixin | Description
 `mdc-drawer-item-text-ink-color($color)` | Sets drawer list item text ink color.
 `mdc-drawer-item-activated-icon-ink-color($color)` | Sets activated drawer list item icon ink color.
 `mdc-drawer-item-activated-text-ink-color($color)` | Sets activated drawer list item text ink color.
-`mdc-drawer-item-corner-radius($radius)` | Sets the corner border radius of the drawer list item.
+`mdc-drawer-shape-radius($radius, $rtl-reflexive)` | Sets the rounded shape to drawer with given radius size. `$radius` can be single radius or list of 2 radius values for trailing-top and trailing-bottom. Set `$rtl-reflexive` to true to flip radius values in RTL context, defaults to true.
```

Comment:
Remove `$rtl-reflexive` here since IIUC we're intentionally not exposing it for drawer (maybe explain instead that only the side farthest from the edge of the screen has radius applied)